<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditBuku</class>
 <widget class="QMainWindow" name="EditBuku">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="labelTitle">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>571</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 30px;</string>
       </property>
       <property name="text">
        <string>Tambah Buku</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLineEdit" name="lStock">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>ISBN</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Kode Buku</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lKdBuku">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Pengarang</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Judul Buku</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Tersedia</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lISBN">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Kode ISBN 13 Digit&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Harga</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Tahun Terbit</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lJudul"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Stock</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="lPenerbit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Penerbit</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="lTahun">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="lHarga">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="lPengarang"/>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QLineEdit" name="lTersedia">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Asal Buku</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QLineEdit" name="lAsalBuku"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="bEditTambah">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>F1. Tambah</string>
        </property>
        <property name="shortcut">
         <string>F1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bEditSimpan">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>F2. Simpan</string>
        </property>
        <property name="shortcut">
         <string>F2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bEditHapus">
        <property name="text">
         <string>F3. Hapus</string>
        </property>
        <property name="shortcut">
         <string>F3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bEditBatal">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>F12. Batal</string>
        </property>
        <property name="shortcut">
         <string>F12</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lKdBuku</tabstop>
  <tabstop>lISBN</tabstop>
  <tabstop>lJudul</tabstop>
  <tabstop>lPengarang</tabstop>
  <tabstop>lTahun</tabstop>
  <tabstop>lPenerbit</tabstop>
  <tabstop>lHarga</tabstop>
  <tabstop>lStock</tabstop>
  <tabstop>lTersedia</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bEditBatal</sender>
   <signal>clicked()</signal>
   <receiver>EditBuku</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>534</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
